packages/app/src/cli/services/context.test.ts

```diff
@@ -727,7 +729,10 @@ api_version = "2023-04"
     vi.mocked(fetchStoreByDomain).mockResolvedValue({organization: ORG1, store: STORE1})
 
     // When
-    const got = await ensureDevContext(INPUT, DEVELOPER_PLATFORM_CLIENT)
+    const dpc = developerPlatformClient()
```

Comment:
I know `developerPlatformClient` is a bit long, but I'd avoid short variable names like this and acronyms.

Reply:
Sounds good. Another option is to leave the [DEVELOPER_PLATFORM_CLIENT constant](https://github.com/Shopify/cli/pull/3451/files#diff-947ada00886ffeb6daf99bafc0b67815bbca34e92222f899186ea26e35734622L156).

Reply:
I can't leave it constant because we are spying on one of the functions, which means that using the same object each time will cause tests to fail.